aws/curator.yaml

```diff
@@ -168,10 +168,10 @@ spec:
           resources:
             requests:
               memory: "512Mi"
-              cpu: "150m"
+              cpu: "500m"
             limits:
               memory: "1Gi"
-              cpu: "250m"
+              cpu: "768m"
```

Comment:
Do these new values fit? We're running two replicas of prod services, and we have 4 total vCPU.

Reply:
yes they do (prod is reloaded now with these values)